incubator/elasticsearch/values.yaml

```diff
@@ -10,7 +10,6 @@ image:
 
 cluster:
   name: "elasticsearch"
-  kubernetesDomain: cluster.local
```

Comment:
Does this get used anywhere else?

Reply:
I thought I deleted that. Let me double delete :)